telemetry/definitions/commonDefinitions.json

```diff
@@ -369,6 +369,138 @@
+            "allowedValues": ["KeyStrokeCount", "SpecialCharacters", "Enter", "IntelliSenseAcceptance"]
+        },
+
+        {
+            "name": "codewhispererCodeScanJobId",
+            "type": "string",
+            "description": "The ID of the security scan job"
+        },
+        {
+            "name": "codewhispererCodeScanLines",
+            "type": "int",
+            "description": "How many lines of code being sent for security scan"
+        },
+        {
+            "name": "codewhispererCodeScanPayloadBytes",
+            "type": "int",
+            "description": "The payload size in bytes of customer project context sent for security scan"
+        },
+        {
+            "name": "codewhispererCodeScanTotalIssues",
+            "type": "int",
+            "description": "The number of security issues been detected"
+        },
+        {
+            "name": "codewhispererCompletionType",
+            "type": "string",
+            "description": "Completion Type of the inference results returned from CodeWhisperer model layer",
+            "allowedValues": ["Line", "Block"]
+        },
+        {
+            "name": "codewhispererCursorOffset",
+            "type": "int",
+            "description": "cursor location offset in the editor when invoking CodeWhisperer for recommendation"
+        },
+        {
+            "name": "codewhispererLanguage",
+            "type": "string",
+            "description": "Programming language of the CodeWhisperer recommendation",
+            "allowedValues": ["java", "python", "javascript", "plaintext"]
+        },
+        {
+            "name": "codewhispererLastSuggestionIndex",
+            "type": "int",
+            "description": "The last index of recommendation from a particular response"
+        },
+        {
+            "name": "codewhispererLineNumber",
+            "type": "int",
+            "description": "The line number of the cursor when the event happens"
+        },
+        {
+            "name": "codewhispererModificationPercentage",
+            "type": "double",
+            "description": "The percentage of user modifications on the suggested code"
+        },
+        {
+            "name": "codewhispererPaginationProgress",
+            "type": "int",
+            "description": "The number of recommendations received so far when user makes a decision"
+        },
+        {
+            "name": "codewhispererPercentage",
+            "type": "int",
+            "description": "The percentage of acceptance on suggested CodeWhisperer code on the overall code"
+        },
+        {
+            "name": "codewhispererRequestId",
+            "type": "string",
+            "description": "The ID of the request to CodeWhisperer service"
+        },
+        {
+            "name": "codewhispererRuntime",
+            "type": "string",
+            "allowedValues": ["java8", "java11", "java16", "python2", "python3", "javascript", "unknown"],
```

Comment:
So the "unknown" here is automatically added by VSC codegen I guess? JB's PR does not have it and VSC's PR has it.

This means, assuming VSC was using the value "unknown" and JB was not using it, adding the unknown here will just make VSC work "as before" and, for JB it's just an extra unused field, if I am understanding this correctly.

Reply:
`unknown` was manually added because VS Code Toolkit isn't automatically doing this.
I was more just curious if the JB Toolkit builds correctly with the redundant value.

Reply:
I did `includeBuild("/Users/yuxqiang/workplace/aws-toolkit-common/telemetry/jetbrains")` in `settings.gradle.kts`
and ran CodeWhispererer telemetry tests and all tests passed.